package.json

```diff
@@ -32,6 +32,14 @@
     "test-multibrowser": "grunt test:multibrowser",
     "test-snyk": "snyk test"
   },
+  "files": [
+    "bin",
+    "build",
+    "lib",
+    "index.js",
```

Comment:
...and since `main` property points to `./build/index.js`, it would be safe to remove `index.js` from the list too.